package.json

```diff
@@ -39,8 +39,8 @@
     "lodash": "^4.17.21",
     "query-string": "^6.11.1",
```

Comment:
can we make a version update for this package, to 2.0.0?

Reply:
yup, will do